### ISSUE TITLE
CMake 3.27 warns about compatibility with CMake <3.5

### ISSUE THREAD

**Issue by kostasdizas** [open-source-parsers/jsoncpp#1521]
**Describe the bug**
CMake 3.27 released earlier this year drops compatibility with CMake <3.5. `jsoncppConfig.cmake.in` requires version 3.0. Using it in a project produces a deprecation warning:
```CMake Deprecation Warning at /usr/local/lib/cmake/jsoncpp/jsoncppConfig.cmake:2 (cmake_policy):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.
```

**To Reproduce**
1. Using CMake 3.27, 

**Expected behavior**
The project is configured without any warnings.

**Desktop (please complete the following information):**
 - OS: Linux
 - Meson version: N/A
 - Ninja version: N/A

**Comment by daudrain** [comment]
Looks to be addressed by #1499

**Comment by daudrain** [comment]
@baylesj This issue can be marked as resolved since #1499 has been merge into master.

**Comment by oold** [comment]
Anybody going to close this issue?

**Comment by xuhdev** [comment]
This has become a problem now for CMake 4.0, see #1671.

The fix is bumping the lower end, not increasing the upper end.

**Comment by baylesj** [comment]
Yes, the lower end needed to be bumped as well, weirdly enough. I don't think it should have to be, but JsonCpp now has a min policy version of 3.5.
